developer_docs/public_api_proposed.md

```diff
@@ -1,12 +1,12 @@
 # Proposed Public API extensions
 
-This describes proposed extensions to the Public API. None of these extensions are confirmed, but are recorded here for reference and discussion. 
+This describes proposed extensions to the Public API. None of these extensions is confirmed but is recorded here for reference and discussion. 
```

Comment:
I think the verb "are" is referring to "extensions". Since "extensions" is plural, I think "are" could be kept!